### DISCUSSION TITLE
TS noUnusedLocals instead of Eslint no-undef  gives false positive?

### DISCUSSION THREAD

**Discussion by iamrgroot** [vuejs/language-tools#510] [Q&A] [answered]
So the recommended way to prevent unused locals with script setup is via TS noUnusedLocals: https://github.com/johnsoncodehk/volar/issues/47

However I'm getting this TS error when I only write to a variable (and read in another component). Simple example: 

![image](https://user-images.githubusercontent.com/8398505/133968037-bcb0b7de-d08a-4e01-a874-1ec917755a8f.png)

What is the recommended way to solve this? Or should I create an issue for this?

I'm using the latest version of Volar with takeover-mode enabled

**Comment by johnsoncodehk** [comment, marked as answer]
The reason for the error is that the user has only been assigned a value, but no value has been taken, so it is not considered to have been used.
